DANS-KNAW/easy-manage-deposit
# Easy 1729 return directory name (uuid) when deposit.properties is not found

- [x]  manual test on deasy

- [x] depositerId? => will be null when deposit.properties does not exist

Fixes Easy-1729 no depositId in full report when deposit.properties is missing 

#### When applied it will
*  show the path to the deposit instead of null, when deposit.properties is not written to disc  properly 

#### Where should the reviewer @DANS-KNAW/easy start?

nl.knaw.dans.easy.managedeposit.EasyManageDepositApp#collectDataFromDepositDir

#### How should this be manually tested?

Add a few deposits without deposit.properties or deposit.properties without an id.
Afterwards run the command for the full report and check if the entires have the filePath as identifier
